assets/js/buttons-examples.js

```diff
@@ -59,7 +59,7 @@ $(document).ready(function () {
           label: "Gradient",
           classes: "gradient-btn",
         },
-       
+
         {
```

Comment:
@RakshitaTantry28 You have to add your button to the buttons array for it to displayed on the website.

Reply:
Okay sure will do the changes.
Could you @HADES-01 please let me know if i have to pull request again after doing changes or it is changed throughout after pushing the code to the branch ?

Reply:
@RakshitaTantry28 No you don't need to make a new PR for this.

Reply:
@HADES-01 then how the changes i have made would get reflected?